src/AlkemioClient.ts

```diff
@@ -24,7 +24,7 @@ import { KratosPublicApiClient } from './util/kratos.public.api.client';
 export class AlkemioClient {
   public apiToken: string;
   public config!: ClientConfig;
-  private client!: Sdk;
+  public client!: Sdk;
```

Comment:
I think this shouldn't be exposed, it should stay private.

Reply:
I considered this. Actually it makes complete sense to have it exposed - as otherwise the steps are to (a) define the queries / mutations (b) run codegen (c) wrap the generated code. Step c was not adding any value as far as I could see...